### PR TITLE
Implement Portfolio Accessibility

### PR DESCRIPTION
**Pull request by Ntare22**
* Page titles - changes made on `line 7` on the index.html pages
* Image text alternatives - no issues found
* Text headings - made change on `line 54 - 56` to create h1
* Color contrast - no issues found
* Resize - no issues found
* Interaction - no issues found
* Moving content - no issues found
* Multimedia - no issues found
* The basic structure of the page - no issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -51,9 +51,9 @@
...
         <section class="text-container">
           <div class="intro">
-            I'm Jim
+            <h1>I'm Jim
             <br />
-            Glad to see you!
+            Glad to see you!</h1>
           </div>
           <p class="desc">
             I'm a Software Developer! I can help you build a product, feature or
```
Comment (carlos-ssh) [inline index.html]: - Provide hover effects to your buttons. With the requested design guidelines in Figma (color and fonts).

File: index.html [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Jim Ntare | Software Engineer</title>
+    <title>Jim Ntare's Home Page</title>
```
Comment (carlos-ssh) [inline index.html]: **Better page titles:**
_Provide a title like these examples._
- Jim's portifolio
- Jim Ntare's full-stack webdeveloper

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Ntare22 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
```
Comment (ZeenatLawal) [inline index.html:4]: - [OPTIONAL] Kindly add the hover effect to the `Get in touch` button in the contact section as requested in the Figma design guidelines.

**Review by ZeenatLawal — APPROVED** [review]
Hi @Ntare22, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply pointed out by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**